src/middleware/rateLimiter.ts

```
@@ -239,4 +301,4 @@ export const injectFallbackState = (
 };
 
 // Export for testing
-export { circuitBreaker, fallbackMetrics, FALLBACK_MAX_REQUESTS_PER_IP, fallbackRateLimitStore };
+export { circuitBreaker, fallbackMetrics, FALLBACK_MAX_REQUESTS_PER_IP, fallbackRateLimitStore };
```

ESLint

[error] Replace `·circuitBreaker,·fallbackMetrics,·FALLBACK_MAX_REQUESTS_PER_IP,·fallbackRateLimitStore·};` with `⏎··circuitBreaker,⏎··fallbackMetrics,⏎··FALLBACK_MAX_REQUESTS_PER_IP,⏎··fallbackRateLimitStore,⏎};⏎`

(prettier/prettier)